app/models/user.rb

```diff
@@ -1,24 +1,27 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great job working on this milestone. As per project requirements, devise is supposed to ask for confirmation of email. In that light, `:confirmable` ought to be enabled in the user model.
![image](https://github.com/Progress2002/Blog-App/assets/62849872/6db42464-7a9d-4697-8f5e-0e3d596a1604)

Reply:
Hello @t-yanick, thanks for the observation, the confirmable feature has been added. Thanks.